docs/whats-new.mdx

```diff
@@ -14,6 +14,10 @@ You can expect this page to update regularly (at least monthly). We'll include t
 
 ## February 2024
 
+- 2024-02-29 - Added high contrast options to the ngrok Dashboard. Look for the brightness icon in the top left of the page when you log in.
```

Comment:
Added High Contrast Light and High Contrast Dark modes to the ngrok Dashboard for increased accessibility. Look for the brightness icon in the top left of the page when you log in.

Reply:
```suggestion
- 2024-02-29 - Added High Contrast Light and High Contrast Dark modes to the ngrok Dashboard for increased accessibility. Look for the brightness icon in the top left of the page when you log in.
```